components/Header.tsx

```diff
@@ -1,39 +1,70 @@
 'use client';
-import { usePathname } from 'next/navigation';
-import Link from 'next/link';
-import Image from 'next/image';
 import useMediaQuery from './hooks/useMediaQuery';
+import { useConversationContext } from '@/context/conversationContext';
+import { usePathname } from 'next/navigation';
+import { useEffect } from 'react';
 
 // Components
```

Comment:
```suggestion
```

while we're here, why not get rid of a needless comment? 🤷

Reply:
I just learned this trick @nichgalzin - apparently you can just accept this suggestion and commit the change from here 🤯

Reply:
This is actually a new convention we are going to try and adopt going forward to make it clear which components are imported from our own code and what's from outside libraries.